### PR TITLE
Update dependencies and code style with Prettier

### PR DESCRIPTION
**Pull request by Amdrel**
Resolves Roblox/roact-performance-benchmarks#91
Resolves Roblox/roact-performance-benchmarks#92

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,5 +41,18 @@
   "bugs": {
     "url": "https://github.com/dead/typeflex/issues"
   },
-  "homepage": "https://github.com/dead/typeflex#readme"
+  "homepage": "https://github.com/dead/typeflex#readme",
+  "prettier": {
+    "tabWidth": 2,
+    "printWidth": 120,
```
Comment (ktrz) [inline package.json]: can we add `"singleQuote": true` to avoid unnecessary changes? It seemed the repo stuck to single quotes before.

Comment (ktrz) [reply package.json]: What do you think of putting the Prettier configuration in a separate file? E.g. `.prettierrc`

**Review by ktrz — COMMENTED** [review]
just some minor comments

**Comment by dustinsgoodman** [comment]
Closes https://github.com/Roblox/roact-performance-benchmarks/issues/91
Closes https://github.com/Roblox/roact-performance-benchmarks/issues/92

**Review by ktrz — APPROVED** [review]
LGTM :shipit:
